Tools/Frame_params/Sub/bluerov2-heavy-3_5_2.params

```diff
@@ -216,11 +216,6 @@ FS_TEMP_ENABLE,0
 FS_TEMP_MAX,62
 FS_TERRAIN_ENAB,0
 GCS_PID_MASK,0
-GND_ALT_OFFSET,0.000000000000000000
-GND_EXT_BUS,1
```

Comment:
GND_EXT_BUS is 1 here, not -1. Shoun't that be preserved?

Reply:
the sub code overrides it